### DISCUSSION TITLE
Is it possible to target more than one country in the Targeting section when adding a banner?

### DISCUSSION THREAD

**Discussion by HazemElknawy9** [giuliopons/adadmin#113] [General]
Is it possible to target more than one country in the Targeting section when adding a banner?
<img width="456" alt="Capture" src="https://github.com/giuliopons/adadmin/assets/64305485/a58b9c18-47e6-42a1-91ce-391e60155c20">

**Comment by giuliopons** [comment]
No. At this moment this is not possible.
I've added a one-time tier in the [sponsor page](https://github.com/sponsors/giuliopons), if someone sponsors this activity I will add multi-country targeting.
